src/styles/blocks/main.scss

```diff
@@ -0,0 +1,22 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
The use of fixed column widths (200px) might cause layout issues on smaller screens. Consider using responsive units or media queries to adjust the layout for smaller devices.